tests/ha/drbd_passive.pm

```diff
@@ -87,7 +87,8 @@ sub run {
     barrier_wait("DRBD_CREATE_CONF_$cluster_name");
 
     # Create the DRBD device
-    assert_script_run "drbdadm create-md $drbd_rsc";
+    my $force = get_var('USE_SUPPORT_SERVER') ? '' : '--force';    # force the md creation if reusing luns
+    assert_script_run "drbdadm create-md $force $drbd_rsc";
```

Comment:
IMHO this `--force` option can be added in any case, this will not hurt on new LUN but can simplify code.

Reply:
OK. Will do that.